lib/jekyll/page.rb

```diff
@@ -1,16 +1,16 @@
 module Jekyll
-  class Page
-    include Convertible
-
+  class Page < Document
     attr_writer :dir
-    attr_accessor :site, :pager
+    attr_accessor :pager
     attr_accessor :name, :ext, :basename
-    attr_accessor :data, :content, :output
+    attr_writer :data
 
     alias_method :extname, :ext
 
     FORWARD_SLASH = '/'.freeze
 
+    # Compatibility
```

Comment:
I've added "Compatibility" flags to methods that are mainly there for compatibility reasons (i.e. removing them shouldn't break core features, only method calls, etc.)